code/modules/surgery/organs/organ_internal.dm

```diff
@@ -76,6 +76,8 @@
 	return ..()
 
 /obj/item/organ/internal/proc/prepare_eat()
+	if(robotic >= 2)
```

Comment:
Might just want to have this be `if(robotic)` so that you can't eat assisted organs either.